docker-compose.yml

```diff
@@ -1,12 +1,16 @@
 version: '3.8'
 services:
-  jupyter:
+  jupyter101:
+    user: root
     build:
-      context: .
-      dockerfile: ./notebooks/Dockerfile
+      context: ./docker
+      dockerfile: Dockerfile
     ports:
-      - '8888:8888'
+      - '8777:8888'
```

Comment:
I believe this maps port 8777 in the container to port 8888 on the host. Is this intended?